### ISSUE TITLE
Color Prefixes to Text of Smash Ranks

### ISSUE THREAD

**Issue by Twillight0420** [SmashMC-Development/Suggestions#1080]
**Describe the suggestion**
I suggest adding an option to Smash ranks in the server to be able to set their text color with a command like /colorgui where they can pick from all the Minecraft color codes. I think it's a good idea to customize your "appearance" in the chat.

**Which servers does this apply to?**
All Pixelmon servers.

**Additional context**
N/A

**Comment by xSire** [comment]
I also agree, it used to be part of the rank, they removed it after the new 1.16 update tho.
